prod-stack.py

```diff
@@ -698,6 +705,7 @@
                 ],
                 'secrets': [
                     'XKAN_GHSECRET',
+                    'DISCORD_WEBHOOK_ID', 'DISCORD_WEBHOOK_TOKEN',
                 ],
```

Comment:
This doesn't need adjustment for me to merge, but we could add these as default secrets. Something like:

```python
secrets = [
    'DISCORD_WEBHOOK_ID', 'DISCORD_WEBHOOK_TOKEN',
    *container.get('secrets', [])
]
```
One less thing to think about when adding containers or services.